### ISSUE TITLE
Auto generated TOC doesn't work on 0.11.13

### ISSUE THREAD

**Issue by TobseF** [BoostIO/Boostnote#3283]
# Current behavior

Boostnote should generate a table of contents on the fly by adding a `[TOC]`.
This feature should be present in Release **0.11.13**: 
[Suppport auto generating toc content for the [TOC] tag #3173](https://github.com/BoostIO/Boostnote/pull/3173)

But I tried it in different ways with `[TOC]` or `[TOC]` with different settings. It never worked. Is there a setting I missed, or a special way to enable it?

![boostnote-toc](https://user-images.githubusercontent.com/1190109/67047420-90b5c080-f132-11e9-90b5-82408a43cec4.gif)

Simply put a  `[TOC]` in the document and add some headings beneath. The side by side preview should show the auto generated TOC.

The dev console doesn't show any error or warning.

**Comment by Flexo013** [comment]
I think you mean version 0.13.0, not 0.11.13.

That being said it seems that this feature is not working in the latest release. I'm using Ubuntu 16.04.

**Comment by Flexo013** [comment]
@hikerpig Could you confirm this feature is missing on your end when using the latest release as well?

**Comment by hikerpig** [comment]
Testing 0.13.0 on OSX 10.15,  works well.

![2019-10-18 18 39 26](https://user-images.githubusercontent.com/2259688/67088009-ae3c6600-f1d6-11e9-88ae-4503d47172fc.gif)

**Comment by Flexo013** [comment]
Thanks for the gif!

@TobseF What OS are you on?

**Comment by TobseF** [comment]
OK, sorry for the confusion. On my computer at work, all works fine with version `0.13.0`. 
So I mixed up the version numbers. Now I only have to check why the auto update feature doesn't work. at home

**Comment by Flexo013** [comment]
The issue still seems to exist for me:

![image](https://user-images.githubusercontent.com/4507015/67091265-7ddfd200-f1ac-11e9-93f9-0e94b53d6e00.png)

![image](https://user-images.githubusercontent.com/4507015/67091280-8801d080-f1ac-11e9-9d28-4f5944aabf2b.png)

**Comment by MrBMT** [comment]
@TobseF there was a bug that broke the auto-update a while back, so that might be why you're not on the latest version at home.

**Comment by arcturus140** [comment]
maybe it only doesn't work in split view?

**Comment by Flexo013** [comment]
The identical note works fine on Windows 10. So it's definitely a Linux only problem.

**Comment by SchrodingersDice** [comment]
I'd just like to ask if the internal linking to the headers are working for you guys?

**Comment by Flexo013** [comment]
Yes it is.

**Comment by SchrodingersDice** [comment]
> 
> 
> Yes it is.

Doesn't seem to be working for me.  Thanks though.
